packages/frontend/src/pages/renderPages.ts

```diff
@@ -107,10 +104,5 @@ export function renderPages(config: Config, pagesData: PagesData) {
     )
   }
 
-  if (config.features.zkCatalog && verifiersApiResponse) {
```

Comment:
I guess zkCatalog feature flag can be removed now.

Reply:
https://github.com/l2beat/l2beat/pull/4378/commits/cc03d5cb18c0e70e4c14881de31e23e63a33e116